src/sdk/model/vapp-template/vapp-template.ts

```diff
@@ -1,15 +1,17 @@
+import { Task } from '..';
```

Comment:
bad import

Reply:
I'll fix it. But... it's generated like this by IDE because Task is exported by index.ts file in the parent directory... So we'll have to check import when we are in this case.